experimenter/experimenter/changelog/views.py

```diff
@@ -1 +1,27 @@
-# Create your views here.
+from django.shortcuts import get_object_or_404
+from django.views.generic import DetailView
+
+from experimenter.experiments.models import NimbusExperiment
+
+
+class NimbusChangeLogsView(DetailView):
+    model = NimbusExperiment
+    template_name = "changelog/overview.html"
+    context_object_name = "experiment"
+
+    def get_object(self, queryset=None):
+        slug = self.kwargs.get("slug")
+        experiment = get_object_or_404(NimbusExperiment, slug=slug)
+        return experiment
```

Comment:
This method is unnecessary. By default, `get_object()` will grab the `slug` from `self.kwargs` and look up the object that way.

[src](https://github.com/django/django/blob/ddb6506618ea52c6b20e97eefad03ed847a1e3de/django/views/generic/detail.py#L40)

[docs](https://docs.djangoproject.com/en/4.2/ref/class-based-views/mixins-single-object/#django.views.generic.detail.SingleObjectMixin.get_object)